easybuild/easyconfigs/f/FRUIT_processor/FRUIT_processor-3.4.3-intel-2018a-Ruby-2.5.1.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'FRUIT_processor'
+version = '3.4.3'
+
+rubyver = '2.5.1'
+versionsuffix = '-Ruby-%s' % rubyver
+
+homepage = 'https://fortranxunit.sourceforge.io'
+description = "FORTRAN Unit Test Framework (FRUIT)"
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://sourceforge.net/projects/fortranxunit/files/fruit_%(version)s']
+
+sources = ['fruit_%(version)s.zip']
+checksums = ['01c88d0b8046d809ab317c7b7e4a87ec343fe1c065d408029d83342808929862']
+
+builddependencies = [('Ruby-Tk', '0.2.0', versionsuffix, toolchain)]
+
+dependencies = [('Ruby', rubyver)]
+
+skipsteps = ['configure', 'build']
+
+preinstallopts = 'GEM_HOME=%(installdir)s '
+preinstallopts += 'gem install --bindir %(installdir)s/bin --local fruit_processor_gem/pkg/fruit_processor-3.4.3.gem ||'
```

Comment:
@SimonPinches This is quite hackish due to the trailing `||` and using `ConfigureMake` rather than `RubyGem`; it seems to highlight a limitation in the `RubyGem` that you're dancing around?

Reply:
@boegel You've probably right but I'm just happy to have a working EB installation :-)

Reply:
OK it's fine if you're not up for this yourself, someone else can pick up on it...

Reply:
Thanks :-)